13.0/base_requirements.txt

```
@@ -0,0 +1,86 @@
+# Odoo dependencies
+Babel==2.3.4
+chardet==3.0.4
+decorator==4.0.10
+docutils==0.12
+ebaysdk==2.1.5
+feedparser==5.2.1
+gevent==1.1.2 ; sys_platform != 'win32' and python_version < '3.7'
```

```suggestion
gevent==1.3.4 ; sys_platform != 'win32' and python_version >= '3.7'
```
We want the 3.7 lib version.